pkg/upstream/replicated.go

```diff
@@ -453,12 +453,23 @@ func listPendingChannelReleases(replicatedUpstream *ReplicatedUpstream, license
 	urlValues := url.Values{}
 	urlValues.Set("channelSequence", sequence)
 	urlValues.Add("licenseSequence", fmt.Sprintf("%d", license.Spec.LicenseSequence))
-	urlValues.Add("downstreamChannelSequence", downstreamCursor.Cursor)
 
-	if downstreamCursor.ChannelID != "" {
-		urlValues.Add("downstreamChannelID", downstreamCursor.ChannelID)
-	} else {
-		urlValues.Add("downstreamChannelName", downstreamCursor.ChannelName)
+	// reporting info
+	if reportingInfo != nil {
+		urlValues.Add("k8sVersion", reportingInfo.K8sVersion)
+		urlValues.Add("isKurl", fmt.Sprintf("%t", reportingInfo.IsKurl))
+		urlValues.Add("appStatus", reportingInfo.AppStatus)
+		urlValues.Add("clusterId", reportingInfo.ClusterID)
+		urlValues.Add("instanceId", reportingInfo.AppID)
```

Comment:
sending this as instanceId instead of appId so that it's not confused with the vendor app id